src/java.net.http/share/classes/jdk/internal/net/http/common/OperationTrackers.java

```diff
@@ -64,6 +64,9 @@ public interface Tracker {
         // Whether the facade returned to the
         // user is still referenced
         boolean isFacadeReferenced();
+        // Whether the implementation of the facade
+        // is still referenced
+        boolean isImplementationReferenced();
```

Comment:
This method appears to be unused

Reply:
Yes - I'm planning to update some tests to use it in some future PRs.